net-libs/qxmpp/qxmpp-1.0.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform C++ XMPP client library based on the Qt framework"
+HOMEPAGE="https://github.com/qxmpp-project/qxmpp/"
+SRC_URI="https://github.com/${PN}-project/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc opus +speex test theora vpx"
+
+RDEPEND="
+	dev-qt/qtcore
+	dev-qt/qtnetwork[ssl]
+	dev-qt/qtxml
+	opus? ( media-libs/opus )
+	speex? ( media-libs/speex )
+	theora? ( media-libs/libtheora )
+	vpx? ( media-libs/libvpx:= )
+"
+DEPEND="${RDEPEND}
+	test? ( dev-qt/qttest:5 )
+"
+BDEPEND="
+	doc? ( app-doc/doxygen )
+"
+
+src_prepare() {
+	# requires network connection, bug #623708
+	sed -e "/qxmppiceconnection/d" \
+		-i tests/CMakeLists.txt || die "failed to drop single test"
+	sed -e "/qxmppserver/d" \
+		-i tests/CMakeLists.txt || die "failed to drop single test"
+	sed -e "/qxmpptransfermanager/d" \
+		-i tests/CMakeLists.txt || die "failed to drop single test"
+
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_DOCUMENTATION=$(usex doc)
+		-DBUILD_EXAMPLES=OFF
+		-DBUILD_TESTS=$(usex test)
```

Comment:
What about this: https://github.com/qxmpp-project/qxmpp/blob/master/CMakeLists.txt#L21

Reply:
Not present in 1.0 series.